Geometry/TrackerCommonData/plugins/DDTrackerIrregularRingAlgo.cc

```diff
@@ -0,0 +1,280 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Where is ` Geometry/TrackerCommonData/plugins/dd4hep/DDTrackerIrregularRingAlgo.cc`? It is required. All Phase 2 development needs to switch to DD4hep.

Reply:
Or this file has to support both DDD and DD4hep.